app/jobs/delete_ead_job.rb

```diff
@@ -43,7 +43,7 @@ def all
     private
 
     # rubocop:disable Metrics/MethodLength
-    def each(&)
+    def each(&block)
       return enum_for(:each) unless block
```

Comment:
shouldn't this be `block_given?`

Reply:
It seems to work the same either way, but I think the better solution is https://github.com/sul-dlss/stanford-arclight/pull/756